### DISCUSSION TITLE
[Android] Retrofit2 대신, Ktor 적용해보기!!

### DISCUSSION THREAD

**Discussion by 1jeongg** [BDD-CLUB/challenge-every-week-posting#163] [1. 개발 관련 글 모음]
비동기 통신 방식으로는 대표적으로 Retrofit2가 있지만 방학 때 새로운 프로젝트를 하면서 새롭게 `Ktor`를 사용해 봤어요!!
거기에 대한 내용을 담아봤습니다 :)

이전에 작성한 블로그 글들은 마음에 안 들어서 다 비공개 처리해서 텅텅 비어있네요ㅠㅠ 지금부터라도 열심히 해보겠습니다! 화이팅❤️

[Ktor 2.3.3 프로젝트에 적용해보기](https://1jeongg.tistory.com/8)

**Comment by shkisme** [comment]
화이팅입니당~!! 🔥
